textext/base.py

```diff
@@ -811,10 +812,16 @@ def import_group_color_style(self, src_svg_ele):
             for it in self.iter():
                 # Update style
                 it.style.update(color_style_dict)
+
+                # Ensure that simple strokes are also colored if the the group has a fill color
+                if "stroke" in it.style and "fill" in color_style_dict:
+                    it.style.update({"stroke": color_style_dict["fill"]})
```

Comment:
Feels like it can be reduced to single `it.style.update(color_style_dict)` with the `if "stroke" in it.style and "fill" in color_style_dict:` patch outside the loop. 

Side note: `it.style.update({"stroke": color_style_dict["fill"]})` should be written as `it.style["stroke"] = color_style_dict["fill"]`